pricing.css

```diff
@@ -0,0 +1,113 @@
+@import url(
+'https://fonts.googleapis.com/css?family=Raleway'
+);
+
+@import url(
+'https://fonts.googleapis.com/css?family=Open+Sans'
+);
+
+
```

Comment:
Please, delete extra lines (between two styles blocks we should have just one empty line)